### ISSUE TITLE
[Graph] 

### ISSUE THREAD

**Issue by spathare21** [sundeep-co-in/transtats#22]

**Comment by sundeep-co-in** [comment]
I agree. Lets take this in steps. I suppose we shall work on testing steps soon, may be - `manual`, `auto`, `performance` etc. But overall a **good idea**!!

**Comment by sundeep-co-in** [comment]
@spathare21 Can we just proceed doing automation for tabular data?

**Comment by spathare21** [comment]
@sundeep-co-in yes sure will check and proceed with it.
